src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js

```diff
@@ -126,7 +147,22 @@ export default function PackageReview({ page: { header, packageId } }) {
             </div>
           )}
           <br />
-          <h2>Package Details</h2>
+          <div className="row">
+            <h2 className="col-sm-6">Package Details</h2>
+            <div className="col-sm-6 text-right">
```

Comment:
I think on mobile we could move the text bellow and not having it stick to the right

![image](https://user-images.githubusercontent.com/51387119/105054496-eb6d5480-5a26-11eb-8ac7-57d8f20040f1.png)

Reply:
Excellent suggestion. Done.